projects/anomaly-detection-stack/layers/monitoring.yml

```diff
@@ -25,5 +25,4 @@
     - elk-docker
     - grafana-docker
     - riemann-docker
-  tags:
-    - aws-start
+  tags: ['aws-setup', 'aws-start']
```

Comment:
Why this task has two tags and the top one is not tagged? Should the top one be tagged with aws-setup or aws-provision so that you can limit playbook execution?